tools/src/wyvern/target/corewyvernIL/support/InvocationExprGenerator.java

```diff
@@ -28,7 +29,7 @@ public InvocationExprGenerator(Expression receiver, String operationName, GenCon
 	
 	@Override
 	public Expression genExpr() {
-		if (declType instanceof ValDeclType || declType instanceof VarDeclType) {
+		if (declType instanceof ValDeclType || declType instanceof VarDeclType || declType instanceof ConcreteTypeMember) {
```

Comment:
Actually it's not legal to create a FieldGet when the field is a ConcreteTypeMember; the result is not a legal expression and should not (for example) typecheck or do anything meaningful at run time.